tests/foreman/installer/test_installer.py

```diff
@@ -1395,14 +1416,29 @@ def test_capsule_installation(sat_default_install, cap_ready_rhel, default_org):
     assert sat_default_install.api.Capsule().search(
         query={'search': f'name={cap_ready_rhel.hostname}'}
     )[0]
+
+    # no errors/failures in journald
+    result = cap_ready_rhel.execute(
+        r'journalctl -q --no-pager -b -p err -u "pulpcore" -u "foreman" -u "foreman-proxy" -u "httpd" -u "postgresql" -u "pulpcore-api" -u "pulpcore-worker"'
```

Comment:
If I am not wrong, you did not cover all services in this list. I would recommend you store all service names in a list that could be placed in constants and generate the command based on it. The list should ideally contain all services that satellite-maintain checks in its `service status` check.

Reply:
non-blocking ^